docs/source/models.rst

```diff
@@ -1902,6 +1914,25 @@ argument. The following values are supported:
     df = spark_df.withColumn("prediction", pyfunc_udf(struct("name", "age")))
 
 
+If you want to use conda to restore the python environment that was used to train the model,
+set the `env_manager` argument when calling :py:func:`mlflow.pyfunc.spark_udf`.
+
+
+.. rubric:: Example
+
+.. code-block:: py
+
+    from pyspark.sql.types import ArrayType, FloatType
+    from pyspark.sql.functions import struct
+
+    pyfunc_udf = mlflow.pyfunc.spark_udf(
+        "path/to/model", result_type=ArrayType(FloatType()),
+        env_manager="conda"  # Use conda to restore the environment used in training
+    )
```

Comment:
```suggestion
    pyfunc_udf = mlflow.pyfunc.spark_udf(
        spark,
        "path/to/model", 
        result_type=ArrayType(FloatType()),
        env_manager="conda"  # Use conda to restore the environment used in training
    )
```

Reply:
Can we include the following

```python
from pyspark.sql import SparkSession

spark = SparkSession.builder.getOrCreate()
```

in the example?

Reply:
Can we add `import mlflow` so we can copy the code in a script and run it?

Reply:
```python
    from pyspark.sql.types import ArrayType, FloatType
    from pyspark.sql.functions import struct
    from pyspark.sql import SparkSession
    import mlflow

    spark = SparkSession.builder.getOrCreate()
    pyfunc_udf = mlflow.pyfunc.spark_udf(
        spark,
        "path/to/model",
        result_type=ArrayType(FloatType()),
        env_manager="conda",  # Use conda to restore the environment used in training
    )
    df = spark_df.withColumn("prediction", pyfunc_udf(struct("name", "age")))
```

Reply:
Ah, `spark_df` is still undefined.

Reply:
Defining a model would make it even longer. I think this is good enough.